### DISCUSSION TITLE
Something is blocking the element

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#5584] [Q&A] [answered]
now i test the explode demo, and i find something is blocking the element or the  camera in viewport is not correct?
how set can see all parts of the element?
https://www.itwinjs.org/sandboxes/iTwinPlatform/Exploded%20View
![123123](https://github.com/iTwin/itwinjs-core/assets/18022720/112610f0-bc94-4271-ab34-4730d7617f5e)

**Comment by pmconne** [comment, marked as answer]
You are most likely translating the element outside of the iModel's project extents. The view frustum is clipped at the near and far planes by the project extents.
